charts/streams-app/templates/scaled-object.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.autoscaling.enabled }}
+{{- $root := . -}}
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {{ template "streams-app.fullname" . }}-so
+spec:
+  scaleTargetRef:
+    name: {{ template "streams-app.fullname" . }}
+  pollingInterval: {{ .Values.autoscaling.pollingInterval }}
+  cooldownPeriod:  {{ .Values.autoscaling.cooldownPeriod }}
+  minReplicaCount: {{ .Values.autoscaling.minReplicas }}
+  maxReplicaCount: {{ .Values.autoscaling.maxReplicas }}
+  triggers:
+    {{- range .Values.streams.inputTopics }}
```

Comment:
Please also iterate extraInputTopics (it is a map, so only check the values of the map)